test/functional/apps/discover/_field_data_with_fields_api.ts

```diff
@@ -60,27 +63,12 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
 
       it('doc view should show Time and Document columns', async function () {
-        const expectedHeader = 'Time Document';
         const Docheader = await PageObjects.discover.getDocHeader();
-        expect(Docheader).to.be(expectedHeader);
+        expect(Docheader).to.contain('Time');
+        expect(Docheader).to.contain('Document');
       });
 
-      it('doc view should sort ascending', async function () {
-        const expectedTimeStamp = 'Sep 20, 2015 @ 00:00:00.000';
-        await PageObjects.discover.clickDocSortDown();
-
-        // we don't technically need this sleep here because the tryForTime will retry and the
-        // results will match on the 2nd or 3rd attempt, but that debug output is huge in this
-        // case and it can be avoided with just a few seconds sleep.
-        await PageObjects.common.sleep(2000);
-        await retry.try(async function tryingForTime() {
-          const rowData = await PageObjects.discover.getDocTableIndex(1);
-
-          expect(rowData.startsWith(expectedTimeStamp)).to.be.ok();
-        });
-      });
-
-      it('a bad syntax query should show an error message', async function () {
+      it.skip('a bad syntax query should show an error message', async function () {
```

Comment:
Why did this test get skipped? It looks like it should work just fine.

Reply:
very true, debugging leftover, thx, removed